pydash/pydash_app/dashboard/fetch.py

```diff
@@ -0,0 +1,66 @@
+import requests
+import jwt
+import json
+
+
+def get_details(dashboard_url):
+    """
+    Get details from a deployed flask-monitoring-dashboard
+    :param dashboard_url: The base URL for the deployed dashboard, without trailing slash
+    :return: A dict containing details from the dashboard, or None if the request was unsuccessful
+    """
+    endpoint = 'get_json_details'
+    response = requests.get(f'{dashboard_url}/{endpoint}')
+
+    if response.status_code != 200:
```

Comment:
Maybe this (when not getting a 200 status code) is a good place to log errors (i.e. the raw response object), so we can debug this if it ever breaks in production?

Reply:
Agreed. It might be a good idea to decide on a logging library/framework.

Reply:
See #57 :smile: